### PR TITLE
[STYLE] Make Python Files Adhere To Set Linting Standards

### PR DESCRIPTION
**Pull request by thealphadollar**
As per the [wiki](https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Maintainer-Info), the python code has been made adhering and all style and refactoring has been done.

0. [style] Make python files adhere to yapf guidelines
	- As per the instructions in Maintainer-info wiki, the python files should adhere to the yapf standard
    - https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Maintainer-Info

0. [style] Make gdb_pretty_printers.py adhere to pylint standards
	- `gdb_pretty_printers.py` has been refactored in order to adhere to the pylint standards set in Maintainer-info wiki

### CODE REVIEW

File: vowpalwabbit/gdb_pretty_printers.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
-import gdb
+"""GDB Pretty Printing Module."""
```
Comment (lalo) [inline vowpalwabbit/gdb_pretty_printers.py]: are the comments in this file necessary?

Comment (thealphadollar) [reply vowpalwabbit/gdb_pretty_printers.py]: Yes. These are docstrings and as per good linting standards, each module, method and function should have a docstring.

Comment (jackgerrits) [reply vowpalwabbit/gdb_pretty_printers.py]: This file isn't part of the vowpalwabbit package, and the comments seem quite redundant.

Comment (thealphadollar) [reply vowpalwabbit/gdb_pretty_printers.py]: Okay. I believe I should remove the docstrings then? Shall I?

Comment (jackgerrits) [reply vowpalwabbit/gdb_pretty_printers.py]: Yes

Comment (thealphadollar) [reply vowpalwabbit/gdb_pretty_printers.py]: @lalo @jackgerrits Thanks for the review. The redundant docstrings have been removed.

**Pull request merged**